tests/test_cmdline.py

```diff
@@ -41,7 +42,7 @@ def make_json_settings_dict(**kwargs):
         "custom_mapping": None,
         "output_format": "human_summary",
         "ignore_undeclared": [],
-        "ignore_unused": [],
+        "ignore_unused": sorted(list(default_ignore_unused)),
```

Comment:
No need to explicitly call `list()` when `sorted()` will handle the `set` directly (and return a list in either case).

```suggestion
        "ignore_unused": sorted(DEFAULT_IGNORE_UNUSED),
```